openmmtools/tests/test_integrators.py

```diff
@@ -247,7 +253,76 @@ def test_external_protocol_work_accumulation():
     n_globals = integrator.getNumGlobalVariables()
     names_of_globals = [integrator.getGlobalVariableName(i) for i in range(n_globals)]
     assert('protocol_work' not in names_of_globals), "Protocol work should not be defined."
-
+    del context, integrator
+
+class TestExternalPerturbationLangevinIntegrator(TestCase):
+    def test_protocol_work_accumulation_harmonic_oscillator(self):
+        """Testing protocol work accumulation for ExternalPerturbationLangevinIntegrator with HarmonicOscillator
+        """
+        testsystem = testsystems.HarmonicOscillator()
+        parameter_name = 'testsystems_HarmonicOscillator_x0'
+        parameter_initial = 0.0 * unit.angstroms
+        parameter_final = 10.0 * unit.angstroms
+        for platform_name in ['Reference', 'CPU']:
+            self.compare_external_protocol_work_accumulation(testsystem, parameter_name, parameter_initial, parameter_final, platform_name=platform_name)
+
+    def test_protocol_work_accumulation_waterbox(self):
```

Comment:
I'm running this test locally, which often fails for the `PME CPU on platform CPU` iteration. The difference between the integrator and external measurements of external work differs only by `1.231683e-05 kT`, which is greater than the `1e-07 kT` default for the `assertAlmostEqual()` in `compare_external_protocol_work_accumulation()`. How about we relax the precision to something like `1e-04 kT`?